### PR TITLE
New contirbution guidelines

### LINKED ISSUE #1: Use Tox to simplify testing and linking
Set up a Tox config to automatically run tests against all supported Python configurations either locally or in CI

### PR DESCRIPTION
**Pull request by AKuederle**
Expanded guidelines with explanation of virtualenv, and tox.

Would close #1 , #2

### CODE REVIEW

**Comment by AKuederle** [comment]
Can you guys have a read through the new text?

The CI will fail for this, beacuse I havn't fixed the existing linting errors. But I will merge it anyway....

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,72 +1,164 @@
...
+```
+python -m unittest discover
+```
+
+## Development Workflow
+
+### Git
+
+In general the [GitFlow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) concept is used for this project.
+This means, that if you want to develop a new feature you need to do so in a new git branch which is based on the `devel` branch or the repository.
```
Comment (BlitzKraft) [inline CONTRIBUTING.md]: I think you meant to say "of the repository".

**Comment by AKuederle** [comment]
Fixed the typo! Thanks @BlitzKraft!
And I just added stuff. The Git section is basically what you wrote just in a text instead of bullet points

**Comment by AKuederle** [comment]
I would merge this if there are no further comments

**Pull request merged**